test-requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --allow-unsafe --generate-hashes --output-file=test-requirements.txt test-requirements.in
+#    pip-compile --generate-hashes --output-file=test-requirements.txt test-requirements.in
```

Comment:
is there a reason why `--allow-unsafe` in the pip-compile command here? We use that parameter to pin `pip` and `setuptools` in the requirements files.

Reply:
confirmed out of band with @creviera that this is not strictly required.